### PR TITLE
new amazing home page

### PR DESCRIPTION
**Pull request by lucien-george**
## Pain
The homepage we had weas really ugly this is a major step up
Screenshot 2021-08-23 at 10 28 32 am

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,5 +1,8 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
-<% if user_signed_in? %>
-  <%= current_user.email %> is currently logged in
-<% end %>
+<div class="container">
+  <h1>Home</h1>
+  <% if user_signed_in? %>
```
Comment (Annedj) [inline app/views/pages/home.html.erb:3]: Clever use of this method! 👌

**Review by Annedj — APPROVED** [review]

**Comment by Annedj** [comment]
Looks good to me 🙌

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,5 +1,8 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
-<% if user_signed_in? %>
-  <%= current_user.email %> is currently logged in
-<% end %>
+<div class="container">
```
Comment (lucien-george) [inline app/views/pages/home.html.erb:1]: nice touch

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+  <h1>Home</h1>
```
Comment (lucien-george) [inline app/views/pages/home.html.erb:2]:
```suggestion
  <h1>Home Page</h1>
```

**Review by lucien-george — COMMENTED** [review]
Nice work ... please see those changes before merging

**Pull request merged**